dev_guide/limits.adoc

```diff
@@ -47,25 +105,38 @@ Per container, the following must hold true if specified:
 |`*Min*`
 |Min[resource] <= container.resources.requests[resource] (required) <= container/resources.limits[resource] (optional)
 
+If the configuration defines a `min` CPU, then the request value must be greater
+than the CPU value. A limit value does not need to be specified.
+
 |`*Max*`
 |container.resources.limits[resource] (required) <= Max[resource]
 
+If the configuration defines a `max` CPU, then you do not need to define a
+request value, but a limit value does need to be set that matches the maximum
+CPU value.
+
 |`*MaxLimitRequestRatio*`
 |MaxLimitRequestRatio[resource] <= ( container.resources.limits[resource] / container.resources.requests[resource])
 
+If a configuration defines a `maxLimitRequestRatio` value, then any new
+containers must have both a request and limit value. Additionally, the
+`maxLimitRequestRatio` value must equal the limit value divided by the
```

Comment:
must equal is not correct.  the limit value divided by the request value must be less than the maxLimitRequestRatio.